test/src/unit-sparse-global-order-reader.cc

```diff
@@ -765,6 +765,97 @@ TEST_CASE_METHOD(
   CHECK(!std::memcmp(data_c, data_r, data_r_size));
 }
 
+TEST_CASE_METHOD(
+    CSparseGlobalOrderFx,
+    "Sparse global order reader: qc removes tile from second fragment "
+    "replacing data from first fragment",
+    "[sparse-global-order][qc-removes-replacement-data]") {
+  // Create default array.
+  reset_config();
+  create_default_array_1d();
+
+  bool use_subarray = false;
```

Comment:
Fixed all but use_qc... Query condition is the point of this fix...

Reply:
Definitely, I just suggested it so that we are sure that we don't introduce any regressions to the other scenarios since I thought that the changes touch code executed in all cases. If you are confident enough, that works for me, I am usually not at all so prefer to over-test 😄